frontend/js/actions/ArimaaActions.js

```diff
@@ -56,7 +142,17 @@ var ArimaaActions = {
         result: data.result
       });
     }
+
+    //TODO sleep 200ms
```

Comment:
Hehe, thanks for the comment. Yes, this TODO is a bit unclear - it's a reminder to myself about a location where there's a hardcoded delay time, becuase at some point, I'm going to do a pass to factor out all timeouts and delays in the code to a central location rather than having hardcoded magic numbers.